CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ project(Natron
     LANGUAGES CXX C
 )
 
-set(CMAKE_CXX_STANDARD 11)
```

Comment:
That's a big change, which may be unnecessary.
For which feature exactly is C++14 required?

Reply:
Some of the C++ algorithm library functionality needs the standard to be 14, but from what I see from local tests and GitHub actions C++14 is compatible with Qt4, so there are no problems as of now.

Reply:
you're right, and actually I'm compiling the plugins in C++14 mode (openimageio requires C++14), and I'm compiling Natron on macOS with C++14 too